src/NoteMouseHandler/NoteMouseHandler.js

```diff
@@ -36,62 +34,54 @@ export default class MouseHandler {
       registerMouseDown,
       registerMouseMove,
       registerMouseUp)
-    actionMouseDown(e, ctx)
+    actionMouseDown(local)
   }
 
-  onMouseMove(e, ctx) {
+  onMouseMove(local, ctx, e) {
     if (this.action) {
-      this.actionMouseMove(e, ctx)
+      this.actionMouseMove(local, e)
     } else {
       let cursor = "auto"
       if (this.cursorHandler) {
-        cursor = this.cursorHandler(e, ctx)
+        cursor = this.cursorHandler(local, ctx)
       }
-      this.emitter.trigger("change-cursor", cursor)
+      ctx.changeCursor(cursor)
     }
   }
 
-  onMouseUp(e, ctx) {
+  onMouseUp(local) {
     if (this.action) {
-      this.actionMouseUp(e, ctx)
+      this.actionMouseUp(local)
     }
     this.action = null
   }
 }
 
-export function defaultActionFactory(e) {
+export function defaultActionFactory(local, ctx, e) {
```

Comment:
__[jshint]__
*  'export' is only available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->